contributing-guides/maintainers-guide.md

```diff
@@ -94,8 +99,7 @@ as a guideline for current and future maintainers.
   then prefer squash, else do a rebase.
 
 - Although having push access allows committing directly to the repository,
-  please **create pull requests for all of your changes**,
-  except the simplest ones (e.g. typo fixes).
```

Comment:
I don't see why not, its just more admin work and pointless reviews.

See one of my typo fix commits [here](https://github.com/tldr-pages/tldr/commit/e4719999325f611503c2ad1dc7bea3e8ac25f557)

Does this really need reviewing? IMO, no because no one is going to reject it and no other changes need to be made.

Reply:
There is always a human factor. So we can make mistake even in fixing typos. It's sounds funny but I am serious. 😁 If we want allow commiting to the main branch without PRs we have to define what kind of typos we can fix in this way. But maybe there are another solutions to this problem. 🤔 I am going to update this file tomorrow.

Reply:
I still recommend opening a PR for trivial changes, even if you merge it yourself right after, though I suppose there are exceptions to every rule.

Reply:
Pushes with trivial changes (formal definition is put above) are allowed now.